android/build.gradle

```diff
@@ -1,36 +1,132 @@
 buildscript {
-    repositories {
-        jcenter()
-        google()
-    }
+  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
+  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['Apptentive_kotlinVersion']
 
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
-    }
+  repositories {
+    google()
+    mavenCentral()
```

Comment:
jcenter() can be removed.

Reply:
There are a few outdated code that need to be cleaned up from `build.gradle`. If releasing is the priority that can be done when updating android new SDK.